js/Audio.js

```diff
@@ -0,0 +1,56 @@
+//The Audio.js is using Audio Web API to create different oscillator type sounds linked to the HMTL buttons.
+var audioContext = new AudioContext();
+
+//Todo: Create one function which will jquery button and based on which button is pressed the type of oscillator will be played
+function playSoundA() {
+    oscillator = audioContext.createOscillator();
+    oscillator.type = 'sine';
+    oscillator.frequency.setValueAtTime(200, audioContext.currentTime);
+    oscillator.connect(audioContext.destination);
+    oscillator.start(audioContext.currentTime);
+    oscillator.stop(audioContext.currentTime + 1);
+};
+
+function playSoundB() {
+    oscillator = audioContext.createOscillator();
+    oscillator.type = 'triangle';
+    oscillator.frequency.setValueAtTime(200, audioContext.currentTime);
+    oscillator.connect(audioContext.destination);
+    oscillator.start(audioContext.currentTime);
+    oscillator.stop(audioContext.currentTime + 1);
+};
+
+function playSoundC() {
+    oscillator = audioContext.createOscillator();
+    oscillator.type = 'square';
+    oscillator.frequency.setValueAtTime(200, audioContext.currentTime);
+    oscillator.connect(audioContext.destination);
+    oscillator.start(audioContext.currentTime);
+    oscillator.stop(audioContext.currentTime + 1);
+};
+
+function playSoundD() {
+    oscillator = audioContext.createOscillator();
+    oscillator.type = 'sawtooth';
+    oscillator.frequency.setValueAtTime(200, audioContext.currentTime);
+    oscillator.connect(audioContext.destination);
+    oscillator.start(audioContext.currentTime);
+    oscillator.stop(audioContext.currentTime + 1);
+};
+
```

Comment:
there's a lot of repeated code here. I would have only include the different code in the functions and initialize the repeated code in another function / in global .

Reply:
Thank you for review. Its my first time. I am sorry that I forgot to share the link: https://codepen.io/Radovan_Jakus/pen/BaopjmN

Yes, you are correct, I have a lot of repetitive code. I would like to reduce it, I was getting out of time.